### DISCUSSION TITLE
zksync plugin issue

### DISCUSSION THREAD

**Discussion by essnuked** [Cyfrin/foundry-full-course-cu#1904] [Q&A] [answered]
So i downloaded the zksync plugin, i compiled my fundme contract and also compiled on the zksync environment but no compiled contract comes in the deploy bar. is anyone having this issues or have a solution?

**Comment by hgky95** [comment]
Your issue is similar to another post, someone has opened an issue on zksync's plugin repo. You can check here https://github.com/NethermindEth/zksync-remix-plugin/issues/124

**Reply by essnuked** [reply]
no solution has been found yet for it

**Reply by hendridg** [reply]
Hi @essnuked 👋, 

Try creating a contracts folder and put the `SimpleStorage.sol` on there 

<img width="421" alt="image" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/73659934/31b5f1ca-c075-44f6-8148-2c512157bc99">

Then make sure your compile version is the same to the contract

<img width="356" alt="image" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/73659934/4555f6fe-f10c-41d5-bee7-32e1f523ccdc">

<img width="261" alt="image" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/73659934/2488cc3f-8169-49de-b786-db684fc07710">


And finally let me know it's was helpful! 😉

**Reply by EngrPips** [reply]
Indeed, This is helpful

**Comment by EngrPips** [comment]
This seems to be a common issue everyone is running into.

**Reply by essnuked** [reply]
yes, it has'nt been fixed yet

**Reply by EngrPips** [reply]
Not yet, I am looking forward to a fix myself. I believe soon there will be a fix for this.

**Reply by jeka0203** [reply]
Hmmm...

**Reply by EngrPips** [reply]
Did you run into the same problem?

**Comment by Insharamin12** [comment]
Seems like others are facing this plugin issue as well. @PatrickAlphaC  
I've been able to compile the contract, but it doesn't show up in the deploy tab. 

Attaching the screenshot here: 

<img width="1439" alt="Screenshot 2024-06-30 at 1 27 07 PM" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/130788673/eb38b31a-f692-442d-a379-cd2a4b58cca2">

**Reply by Sumit0x00** [reply]
same issue with me

**Reply by EngrPips** [reply]
Have you tried to use solidity version `0.8.25`? That seems to solve it for some people.

**Reply by Sumit0x00** [reply]
Yes brother, but now it  solved, I created a directory named contracts and moved the .sol file in it.

**Reply by EngrPips** [reply]
That awesome. seems like we all have a way to bypass this issue.

**Comment by Verifieddanny** [comment]
I used ATLAS IDE to deploy mine since it's not working on Remix

https://zksync.atlaszk.com/ide

**Comment by PatrickAlphaC** [comment]
I am able to reproduce; we are in contact with the team now.

**Reply by EngrPips** [reply]
Interesting, Thanks for all you do, sir.

**Comment by PatrickAlphaC** [comment, marked as answer]
Hi all.

This is an issue with the zksync plugin, I will add another video next week.

The solution is to create a "contracts" folder in Remix and place your contracts into there, and then compile with the zksync plugin. 

Sorry for this issue!

**Reply by EngrPips** [reply]
Excellent. Thanks for all you do, sir.

**Reply by SairajMN** [reply]
Thanks a lot sir.
